src/Common/CommonServices.ts

```diff
@@ -472,14 +481,66 @@ export interface IGlobalMessageBanner {
      * Links to supply to the format arguments in `messageFormat`
      */
     messageLinks?: IGlobalMessageLink[];
+
+    /**
+     * Where the banner should appear.
+     */
+    position?: string;
+}
+
+/**
+ * Toast Props with metadata to specify how it will be rendered by the service
+ */
+export interface IToast {
+    /**
+     * Optional text for the Call to Action
+     */
+    callToAction?: string;
+
+    /**
+     * Optional class name for the root toast element
+     */
+    className?: string;
+
+    /**
+     * Duration in ms the toast will appear for
+     */
+    duration: number;
+
+    /**
+     * If true, we'll immediately take down any existing toast and display this instead
+     * Otherwise, it adds it to an internal queue in the GlobalToast and will display after others in the queue
+     */
+    forceOverrideExisting?: boolean;
+
+    /**
+     * Message to display on the Toast
+     */
+    message: string;
+
+    /**
+     * Optional handler for when the Call to Action is clicked
+     */
+    onCallToActionClick?: () => void;
 }
 
 export interface IGlobalMessagesService {
+    /**
+     * Adds a new message banner to the displayed banners
+     * @param banner - The message banner to display
+     */
+    addBanner(banner: IGlobalMessageBanner): void;
 
     /**
      * Sets the currently displayed global message banner
-     * 
+     * DEPRECATED - COMPAT OLNY - use @see addBanner instead
      * @param banner - The message banner to display
      */
-    setGlobalMessageBanner(banner: IGlobalMessageBanner): void
+    setGlobalMessageBanner(banner: IGlobalMessageBanner): void;
```

Comment:
just go ahead and remove this method from the interface. Consumers of the new API library can switch to the new method.